client-app/src/examples/contact/mobile/AppModel.ts

```diff
@@ -75,17 +72,16 @@ export default class AppModel extends BaseAppModel {
     override async initAsync() {
         await super.initAsync();
         await XH.installServicesAsync(ContactService);
-        this.loadAsync();
+        await this.loadAsync();
     }
 
     override async doLoadAsync() {
         try {
-            this.contacts = await XH.contactService.getContactsAsync();
-
+            const contacts = await XH.contactService.getContactsAsync();
             runInAction(() => {
-                this.tagList = uniq(this.contacts.flatMap(it => it.tags ?? [])).sort() as string[];
+                this.contacts = contacts;
```

Comment:
We need to write observables within an action. Alternatively, we could declare them as @bindable